cinema/views.py

```diff
@@ -167,6 +183,24 @@
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="movie",
+                type={"type": "number"},
+                description="Filter by movie id (ex. ?movie=2)",
+            ),
+            OpenApiParameter(
+                name="date",
+                type={"type": "string", "format": "date"},
+                description="Filter by date in '%Y-%m-%d' format "
+                            "(ex. ?date=2019-02-01)"
+            )
+        ]
+    )
+    def list(self, request, *args, **kwargs):
+        return super().list(request, *args, **kwargs)
+
```

Comment:
The `extend_schema` decorator is also used in `MovieSessionViewSet` to document the `list` action. This helps in providing clear API documentation for filtering movie sessions by movie and date.